### ISSUE TITLE
In onBindViewHolder() the 'drawable instanceof Bitmap' check is always false.

### ISSUE THREAD

**Issue by amaa-99** [amaa-99/afwall#15]
**Describe the bug**
In onBindViewHolder() the 'drawable instanceof Bitmap' check is always false thus leading always to the execution of the else case.

**Firewall Logs**
-

**Smartphone (please complete the following information):**
 - Device: Any
 - Android OS: Any

**Additional context**
-

**Comment by amaa-99** [comment]
There are inconsistencies on the setting of the icon image in that area of the code.
In that spot the following code gets executed:
```
holder.icon.setBackground((Drawable)drawable);
```
but a few lines later:
```
if (Build.VERSION.SDK_INT > android.os.Build.VERSION_CODES.O) {
    holder.icon.setBackground(context.getDrawable(R.drawable.ic_unknown));
} else {
    Drawable appIcon = context.getDrawable(R.drawable.ic_unknown);
    holder.icon.setImageBitmap(Api.getBitmapFromDrawable(appIcon));
}
```
i.e., depending on SDK version it will use setBackground() or setImageBitmap().

**Comment by amaa-99** [comment]
This has been addressed by the fixes implemented for the issue #3.

**Comment by amaa-99** [comment]
This has been addressed by the fixes implemented for the issue https://github.com/amaa-99/afwall/issues/3.
